app/controllers/application_controller.rb

```diff
@@ -71,9 +71,12 @@ def set_back_path
   end
 
   def find_current_local_authority_from_subdomain
-    return if @current_local_authority ||= LocalAuthority.find_by(subdomain: request.subdomains.first)
+    subdomain = request.subdomains.first
 
-    render plain: "No Local Authority Found", status: :not_found
+    return if subdomain == "config"
```

Comment:
There's a way to avoid this palaver in the engine's routes - have a catchall route at the end and return a `x-casade` header with a 404, e.g. 

``` ruby
get "/*path", to: proc { [404, { "x-cascade" => "pass" }, %w[]] }
```

This will trigger the Rails router to stop looking for routes.

Another approach is to wrap the main application routes in a constraint that checks for valid LA subdomains (which is probably a better approach long term).

Reply:
Gone with the wrapping main application routes in a constraint